src/test_lib_json/main.cpp

```diff
@@ -2814,9 +2814,8 @@ JSONTEST_FIXTURE_LOCAL(ReaderTest, strictModeParseNumber) {
 }
 
 JSONTEST_FIXTURE_LOCAL(ReaderTest, parseChineseWithOneError) {
-  // \u4f50\u85e4 佐藤
   checkParse(R"({ "pr)"
-             "佐藤"
+             "\u4f50\u85e4" // 佐藤
```

Comment:
Hi Billy, we have already tried this way, but it failed in Appveyor, you can see this [log](https://ci.appveyor.com/project/cdunn2001/jsoncpp/builds/29243742/job/ijmph5b353ce9a7d). Actually, It has also reported this kind of error when you comment on [this](https://github.com/open-source-parsers/jsoncpp/pull/1095#discussion_r345037318) before.

Reply:
I don't understand that failure though. Do you know what is going on there?
It should be that my change makes NO difference to the compiled bytes.

I believe the \u encodings are correct.

How did this string literal encoding change turn into:

  C:\projects\jsoncpp\src\test_lib_json\main.cpp(2659): a.offset_start == e.offset_start
    Expected: 14
    Actual  : 180
  C:\projects\jsoncpp\src\test_lib_json\main.cpp(2660): a.offset_limit == e.offset_limit
    Expected: 15
    Actual  : 1900
  C:\projects\jsoncpp\src\test_lib_json\main.cpp(2682): reader->getFormattedErrorMessages() == formatted
    Expected: * Line 1, Column 15
EXEC : Syntax error : value, object or array expected. [C:\projects\jsoncpp\src\test_lib_json\jsoncpp_test.vcxproj]    
    Actual  : * Line 1, Column 19
EXEC : Syntax error : value, object or array expected. [C:\projects\jsoncpp\src\test_lib_json\jsoncpp_test.vcxproj]


This is happening on MSVC 14.0 (2015).
I wonder if that's buggy w.r.t. unicode?

Reply:
Something wrong when we use `"\u4f50\u85e4"`, but it is OK if we use `u8"\u4f50\u85e4"`.
I also wonder the reason why it fails?
Anyway,  i push a pr for replacing raw unicode.
PR: https://github.com/open-source-parsers/jsoncpp/pull/1127